src/metrics/stats.js

```diff
@@ -82,7 +82,7 @@ class Stats extends EventEmitter {
   /**
    * Returns a clone of the internal movingAverages
    *
-   * @returns {Object}
+   * @returns {typeof Object.assign}
```

Comment:
weird, can you explain this one?

Reply:
No need for this, it can be inferred automatically. Good catch